spec/support/rspec_rails_monkeypatch_for_rails5.rb

```diff
@@ -0,0 +1,16 @@
+require 'rspec/rails/version'
+module RSpec::Rails
+  module ControllerExampleGroup
+    raise "Remove this" if RSpec::Rails::Version::STRING >= '3'
```

Comment:
Linking https://github.com/ManageIQ/manageiq/pull/6026 here for reference, as this and [this](https://github.com/ManageIQ/manageiq/pull/5982/files#r48670428) may not be needed very soon(?)